[Operational Issue]:  Manifest assets specified twice in SDK band build

### Error

```
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.Current.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.net6.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.net6.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.net7.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.net7.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.net8.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.net8.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.net9.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Emscripten.net9.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.net7.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.net7.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.net8.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.net8.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
##[error].packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.net9.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\11.0.0-beta.25570.6\tools\SdkTasks\PublishArtifactsInManifest.proj(126,5): error : Asset 'Microsoft.NET.Workload.Mono.Toolchain.net9.Manifest-10.0.100-preview.0' is specified twice in the build information. Assets should not be duplicated.
```

### Links

- https://dev.azure.com/dnceng/internal/internal%20Team/_build/results?buildId=2876969&view=logs&j=ada50d30-79d4-567d-7b38-6676fc3bede8&t=49d145cc-0bfb-5d05-f991-7dcc98e76132&l=43

### Root Cause

unknown

### Blocking

- [x] This issue is blocking

### Affected Releases

- [ ] .NET 10
- [x] .NET 10 (SDK band build)
- [ ] .NET 11

--- a/repo-projects/sdk.proj
+++ b/repo-projects/sdk.proj
@@ -21,6 +21,7 @@
     
     <BuildArgs Condition="'$(BuildWorkloads)' == 'true'">$(BuildArgs) /p:BuildWorkloads=true</BuildArgs>
     <BuildArgs Condition="'$(BuildWorkloads)' == 'true' and '$(DotNetBuildSharedComponents)' != 'true'">$(BuildArgs) /p:DownloadWorkloadMsis=true</BuildArgs>
+    <BuildArgs Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(DotNetBuildSharedComponents)' != 'true'">$(BuildArgs) /p:SkipManifestBuild=true</BuildArgs>
     <BuildArgs Condition="'$(DotNetBuildSharedComponents)' != 'true'">$(BuildArgs) /p:DotNet1xxRuntimeVersion=$(MicrosoftNETCoreAppRefVersion) /p:DotNet1xxWorkloadManifestVersion=$(DotNet1xxWorkloadManifestVersion)</BuildArgs>
   </PropertyGroup>
 

--- a/src/sdk/src/Workloads/Manifests/manifest-packages.csproj
+++ b/src/sdk/src/Workloads/Manifests/manifest-packages.csproj
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.Build.Traversal" DefaultTargets="Pack">
 
   <ItemGroup>
-    <ProjectReference Include="**\*.Manifest.proj" />
+    <ProjectReference Include="**\*.Manifest.proj" Condition="'$(SkipManifestBuild)' != 'true'" />
   </ItemGroup>
 
   <Target Name="LayoutBuiltinManifests">

